sdk/CHANGELOG.md

```diff
@@ -25,10 +25,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Migrated storage types field casing to uniform camelCase;
 - Replaced inappropriate serde impls from `AccountDetails` with fn impls for conversion;
+- `MqttPayload` now uses Dtos;
+- `NodeManagerBuilder` node fields are no longer wrapped in `Option`;
+- Prefix-hex string values replaced with boxed slices in Dtos;
```

Comment:
Why?

Reply:
Less conversions I guess

Reply:
Oh was that because Strings were never actually empty because of 0x?

Reply:
Yes, and we would have to add the check for emptiness in our conversion code rather than in serde.